CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 # Upcoming release
 
 ### Added
+- **UILabel**
+    - Added `convenience init(text:style)` to create a `UILabel` with a text and an appropriate font. [#607](https://github.com/SwifterSwift/SwifterSwift/pull/607) by [marcocapano](https://github.com/marcocapano)
```

Comment:
No need to include `convenience` for the signature. Also, the argument isn't a font, that would have been of type `UIFont`, it's more a font _style_. I actually think it would make more sense to pass a `font` instead than a `style`, but if this is useful for you then it's probably good for us to have as well :)
```suggestion
    - Added `init(text:style)` to create a `UILabel` with a text and font style. [#607](https://github.com/SwifterSwift/SwifterSwift/pull/607) by [marcocapano](https://github.com/marcocapano)
```